src/lmic/lmic_config_preconditions.h

```diff
@@ -92,6 +92,11 @@ Revision history:
 # include CFG_TEXT_1(ARDUINO_LMIC_PROJECT_CONFIG_H)
 #endif /* ARDUINO_LMIC_PROJECT_CONFIG_H_SUPPRESS */
 
+#ifdef CFG_au921
```

Comment:
BSPs like the MCCI BSP need to support old and new LMIC. So for now they define both `CFG_au921` and `CFG_915`. So this would be better as:

```c
#if defined(CFG_au921) && !defined(CFG_au915)
```

If we add that, this is ready to merge. It now passes CI testing. Thanks!

Reply:
Oops. It will be a breaking change for older code if we don't also #define CFG_au921 if we see CFG_au915 defined. 

so we also need (after the above):

```c
#if defined(CFG_au915) && ! defined(CFG_au921)
// for backwards compatibility to legacy code, define CFG_au921 if we see CFG_au915.
# define CFG_au921
#endif
```

Reply:
Fixed!